test/adapters/http_server.spec.js

```diff
@@ -276,7 +276,7 @@ describe('read http_server', function() {
             return finish;
         })
         .then(function(results) {
-            expect(results.errors).to.deep.equal(['Invalid CSV data: Error: Row length does not match headers']);
+            expect(results.errors).to.deep.equal(['Invalid CSV data: Red Dragon, 24.04 at line 2']);
```

Comment:
It seems like we lost something useful in the `Row length does not match headers`, can we have both that and the new message?